src/lib/features/feature-toggle/feature-toggle-store.ts

```diff
@@ -467,20 +473,39 @@ export default class FeatureToggleStore implements IFeatureToggleStore {
             stale: data.stale,
             created_at: data.createdAt,
             impression_data: data.impressionData,
+            created_by_user_id: data.createdByUserId,
```

Comment:
Ah, that was a double take, it looked like you were excluding the createdByUserId and then using it. And then I spotted the fact that you changed it from optional to required. Nice.